src/main/js/util/fieldMapping.js

```diff
@@ -13,18 +13,21 @@ import DescriptorOption from 'component/common/DescriptorOption';
 
 function trimValue(items) {
     const { value } = items;
-    const trimmedValue = (Array.isArray(value)) ? value[0] : value;
+    return (Array.isArray(value)) ? value[0] : value;
+}
 
+function insertTrimmedValue(items) {
+    const trimmedValue = trimValue(items);
     return Object.assign(items, { value: trimmedValue });
```

Comment:
If I understand the trimValue method, then this is adding the same value into items that was already there?

Reply:
What happens here is we get the single item from an array or we get the single value already passed to us. Then we insert it into the old items object and overwrite the old value with this new, potentially changed, value.

Reply:
I think we may want to rename this to reduce confusion. overwriteSingleValue?